build.sh

```diff
@@ -218,6 +220,16 @@ test_ios_devices() {
     configuration="$3"
     failed=0
     for device in "${serial_numbers[@]}"; do
+        if [[ "$device" = "07510d12c09c53e44b3682dd0367934f82cfab9c" ]]; then
```

Comment:
You're right, I made this PR before we decided to parallelize the objc and swift device tests. I'll remove this change.